petcare-sample/android-mobile-application/README.md

```diff
@@ -0,0 +1,266 @@
+>    ```
+> 
+> #### Step 3: Refactor the Package Name
+> 
+> 1. Right-click on the old package name in the project explorer.
+> 2. Select "Refactor" > "Rename" (or just press `Shift` + `F6`).
+> 3. Enter the new package name and click "Refactor."
+> 
+> #### Step 4: Update Gradle Files
+> 
+> 1. Open the `build.gradle` file (Module: app).
+> 2. Update the `applicationId` to the new package name.
+> 
+>    ```gradle
+>    android {
+>        ...
+>        defaultConfig {
+>          ...
+>            applicationId "com.your.new.packagename"
+>        }
+>    }
+>    ```
+> 
+> 3. Sync your project with Gradle by clicking on the "Sync Now" prompt that appears in the bar at the top.
+> 
+> #### Step 5: Rename Source Code Directories
+> 
+> 1. Right-click on the old package name directory and choose "Refactor" > "Rename."
+> 2. Enter the new package name and click "Refactor."
+> 
+> #### Step 6: Update References
+> 
+> 1. Android Studio will update most references automatically, but double-check for any manual references in your code or resources.
+> 2. Look for occurrences of the old package name and update them to the new one.
+> 
+> #### Step 7: Clean and Rebuild
+> 
+> 1. Clean your project by selecting "Build" > "Clean Project" from the menu.
+> 2. Rebuild your project by selecting "Build" > "Rebuild Project."
+> 
+> #### Step 8: Test Your App
+> 
+> 1. Run your app on an emulator or a physical device to ensure that everything is working as expected.
+> 2. Check for any errors or warnings in the logcat.
+
+Also make sure to subscribe to Google Play Integrity API from the project, you can do this from `Enabled API's & Services` in the Google cloud project.
+
+6.2 After that you need to create a new service account in the google project.
+You can create a Service Account for yourself with the following scopes.
+- Go to IAM& Admin -> Service Accounts
+- Click Create Service Account
+- Fill the name and click create and continue.
+- You need to grant your service account the roles of Service Account User and Service Usage Consumer.
+- Click continue and then Done
+- You can see the service account added without keys, click : Actions -> Manage Keys for the service account.
+- Click Add key and Select JSON.
+- Save the JSON in secure place (We need this for Android Attestation Credentials for application metadata)
+
+6.3 After that, Update Application Advanced properties.
+The application you created requires 2 properties to perform android attestation.
+- Android package name
+- androidAttestationServiceCredentials
+  - The JSON secret of Service Account downloaded. Note that this attribute is defined as a JSON object hence use the JSON key as it is.
+
+6.4 Go to the Google play console and create a new application for this application, after that you need to associate your cloud project for the app integrity. To do this,
+- Navigate to Release > App integrity. Under Play Integrity API select Link a Cloud project.
```

Comment:
```suggestion
- Navigate to Release > App integrity. Under Play Integrity API, select **Link a Cloud project**.
```

Reply:
Fixed with https://github.com/wso2/samples-is/pull/443/commits/a0de749a7604845679f1995fc79c5815c95f7a5d